src/app/logging/log-table/log-table.component.ts

```diff
@@ -0,0 +1,90 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { LogMessageFormat, LogType } from 'logging-format';
+import { MatTableDataSource } from '@angular/material/table';
+import { MatSort } from '@angular/material/sort';
+import { trigger, state, style, transition, animate } from '@angular/animations';
+import { HttpClient } from '@angular/common/http';
+
+// Dummy Logs for testing purpose
+const dummyLogs: LogMessageFormat[] = [
+  {
+    time: Date.now() + 1000 * 60 * 60 * 2,
```

Comment:
hier wäre eine gute stelle für ein kommentar. eines, das uns mitteilt, was genau die addition von 1000 * 60 * 60 * 2 bezweckt. denn ich bezweifle, dass wir uns daran in einigen wochen oder monaten noch erinnern können.